### PR TITLE
[hotfix] fix dashboard

### PR DESCRIPTION
**Pull request by hzy46**
fix the port issues for k8s dashboard:
image

### CODE REVIEW

File: deployment/k8sPaiLibrary/template/dashboard-service.yaml.template [hunk 1]
```diff
@@ -27,6 +27,3 @@ metadata:
 spec:
   selector:
     k8s-app: kubernetes-dashboard
-  ports:
-  - port: 80
```
Comment (abuccts) [inline deployment/k8sPaiLibrary/template/dashboard-service.yaml.template]: why remove? and you should use `paictl` to deploy instead of `kubectl`

Comment (hzy46) [reply deployment/k8sPaiLibrary/template/dashboard-service.yaml.template]: If it is not removed, an error will be raised when deployed. But the CI shows this should not be removed. I'm trying to figure out why the error happens when I deploy it.

Comment (abuccts) [reply deployment/k8sPaiLibrary/template/dashboard-service.yaml.template]: Please __clean all services and k8s__ before your deployment.
I guess the k8s or dashboard's versions may be changed by others.

/cc @ydye to help.

Comment (hzy46) [reply deployment/k8sPaiLibrary/template/dashboard-service.yaml.template]: I used "./paictl.py service stop" and "./paictl.py cluster k8s-clean" to clean the cluster before I deploy. But the problem remains.  It occurs only on int bed. Another bed doesn't have this issue.

**Comment by hzy46** [comment]
Close this issue since it is not a bug but an environmental problem.

**Pull request closed without merging**